.github/labeler.yml

```
@@ -0,0 +1,14 @@
+🧩 cli: packages/cli/**
+🧩 core: packages/core/**
+🧩 models: packages/models/**
+🧩 nx-plugin: packages/nx-plugin/**
+🧩 eslint-plugin: packages/plugin-eslint/**
+🧩 lighthouse-plugin: packages/plugin-lighthouse/**
+🧩 utils: packages/utils/**
+
+🔬 testing:
+  - packages/**/*.spec.ts
```

The test file pattern changed recently. Also, we could have tests in other folders than `packages` as well.

```suggestion
  - **/*.test.ts
```